src/index.js

```diff
@@ -28,18 +29,15 @@ function resolveConfigPath(filePath) {
 }
 
 const getConfigFunction = config => () => {
-  if (_.isUndefined(config) && !_.isObject(config)) {
```

Comment:
The 2nd part of this conditional is unnecessary. An undefined variable will never be an object.